tests/tests/fv3_ccpp_regional_c768_FA_update_moist

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+#
+#  FV3 CCPP regional c768 test
```

Comment:
You may want to update the description to mention that this is testing the Ferrier-Aligo scheme.